src/plugins/content/accessibility/keyNavigation/strategies/navigatorNavigation.js

```diff
@@ -102,15 +142,19 @@ export default {
                 }
                 if (config.keyPrevContent) {
                     filtersNavigator.on(config.keyPrevContent, elem => {
-                        if (allowedToNavigateFrom(elem) && itemsNavigator) {
+                        if (allowedToNavigateFrom(elem) && testStatusNavigation) {
+                            testStatusNavigation.focus();
+                        } else if (allowedToNavigateFrom(elem) && itemsNavigator) {
                             _.defer(() => {
                                 itemsNavigator.last();
-                            });
+                            })
                         }
                     });
                 }
 
-                this.keyNavigators.push(filtersNavigator);
+                if (!testStatusNavigation || !config.keyNextTab || !config.keyPrevTab) {
```

Comment:
This is preventing the filters to be accessed using the tab key in linear mode.

Reply:
It's ok. In linear mode the user uses tab to travel between strategies (header, topbar, review panel etc) and he should use arrows to switch between elements inside the group. So in this case the filters is accessible by tab if `keyNextTab `and `keyPrevTab` are defined and no `testStatusNavigation `element inside the review panel.